src/ring_theory/algebraic.lean

```diff
@@ -104,4 +106,27 @@ begin
   exact is_integral_trans L_alg A_alg,
 end
 
+/-- A field extension is algebraic if it is finite. -/
+lemma is_algebraic_of_finite (finite : finite_dimensional K L) : is_algebraic K L :=
```

Comment:
```suggestion
lemma is_algebraic_of_finite [finite_dimensional K L] : is_algebraic K L :=
```
Is there a reason you don't want to take a tc argument?

Reply:
The simple answer is: I didn't realize it was a typeclass :P There are some more instances of `finite_dimenional`, let me fix them...